app/views/pages/index.html.erb

```diff
@@ -134,14 +134,15 @@
       <article class="HomepageCategory">
         <%= link_to "/docs/agent/v3/elastic-ci-aws/elastic-ci-stack-overview", class: "HomepageCategory__header" do %>
           <h3 class="HomepageCategory__heading">Elastic CI Stack</h3>
-          <p class="HomepageCategory__subtext">Get started with Buildkite Elastic CI Stack for:</p>
+          <p class="HomepageCategory__subtext">Set up your Buildkite Elastic CI Stack</p>
```

Comment:
style, optional: Again, this might be a personal preference but "your" alongside company/product names feels a little boundary stepping.

```suggestion
          <p class="HomepageCategory__subtext">Set up Buildkite Elastic CI Stack</p>
```